client/components/SearchPage/SearchToolbar.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+import { Nav, NavItem } from 'react-bootstrap'
+import Icon from 'components/Common/Icon'
+
+class SearchToolbar extends React.Component {
+  constructor() {
+    super()
```

Comment:
❓  `SearchToolbar` doesn't catch `props` on constructor, is it okay?

Upper level component passes props though ...
https://github.com/crowi/crowi/pull/312/files#diff-24c2fd48e6e519712edba1bff9de226bR95

Reply:
I fixed this on 53e4c0a